packages/expo/package.json

```diff
@@ -42,6 +42,7 @@
     "styled-components": "5.0.0"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.14.2",
```

Comment:
So instead of adding separated plugins, what do you think about using the preset from expo? 
Check it out: https://github.com/expo/expo/tree/master/packages/babel-preset-expo

It includes `@babel/preset-env` which already includes most plugins out there plus other configs like lazyImports
cc @belgamo

Reply:
LGTM. I would do that in another PR.